include/zephyr/bluetooth/conn.h

```diff
@@ -479,6 +479,50 @@ struct bt_conn_le_tx_power {
 	int8_t max_level;
 };
 
+enum bt_conn_le_tx_power_report_reason {
+	/** Local Transmit power changed. */
+	BT_CONN_LE_TX_POWER_REPORT_REASON_LOCAL_CHANGED = 0,
+
+	/** Remote Transmit power changed. */
+	BT_CONN_LE_TX_POWER_REPORT_REASON_REMOTE_CHANGED = 1,
+
+	/** HCI_LE_Read_Remote_Transmit_Power_Level command completed. */
+	BT_CONN_LE_TX_POWER_REPORT_REASON_READ_REMOTE_COMPLETED = 2,
+};
+
+/** LE Transmit Power Reporting Structure */
+struct bt_conn_le_tx_power_report {
+
+	/** Reason for Transmit power reporting. */
+	enum bt_conn_le_tx_power_report_reason reason;
+
+	/** @ref bt_conn_le_tx_power_phy */
+	uint8_t phy;
+
+	/** Transmit power level
+	 * - 0xXX - Transmit power level
+	 *  + Range: -127 to 20
+	 *  + Units: dBm
+	 *
+	 * - 0x7E - Remote device is not managing power levels on this PHY.
+	 * - 0x7F - Transmit power level is not available
+	 */
+	int8_t tx_power_level;
+
+	/** Bit 0: Transmit power level is at minimum level.
+	 *  Bit 1: Transmit power level is at maximum level.
+	 */
+	int8_t tx_power_level_flag;
+
+	/** Change in transmit power level
+	 * - 0xXX - Change in transmit power level (positive indicates increased
+	 *   power, negative indicates decreased power, zero indicates unchanged)
+	 * Units: dB
```

Comment:
```suggestion
	 * - 0xXX - Change in transmit power level (positive indicates increased
	 *   power, negative indicates decreased power, zero indicates unchanged).
	 *   Units: dB
```

Reply:
waiting on an answer to https://github.com/nrfconnect/sdk-zephyr/pull/1376#discussion_r1383555116 and will update this then :)